test/end-to-end/pages/declaration/declaration.js

```diff
@@ -1,14 +1,22 @@
 'use strict';
 
 const config = require('config');
-const commonContent = require('app/resources/en/translation/common');
-const content = require('app/resources/en/translation/declaration');
+const commonContentEn = require('app/resources/en/translation/common');
+const commonContentCy = require('app/resources/cy/translation/common');
+const contentEn = require('app/resources/en/translation/declaration');
+const contentCy = require('app/resources/cy/translation/declaration');
+const pagePath = require('app/steps/ui/declaration');
 
```

Comment:
This is undoing performance work - including these files takes time - do we need to?